File: Modelo/Proveedor.cs
namespace punto_de_venta_C_.Modelo
{
    public class Proveedor
    {
        private int id;
        private string nombre;
        private string contacto;
        private string telefono;
        private string email;
        private string direccion;
        private bool activo;

        public Proveedor(int id, string nombre, string contacto, string telefono, string email, string direccion, bool activo)
        {
            this.id = id;
            this.nombre = nombre;
            this.contacto = contacto;
            this.telefono = telefono;
            this.email = email;
            this.direccion = direccion;
            this.activo = activo;
        }

        public int getId() { return id; }
        public void setId(int id) { this.id = id; }
        public string getNombre() { return nombre; }
        public void setNombre(string nombre) { this.nombre = nombre; }
        public string getContacto() { return contacto; }
        public void setContacto(string contacto) { this.contacto = contacto; }
        public string getTelefono() { return telefono; }
        public void setTelefono(string telefono) { this.telefono = telefono; }
        public string getEmail() { return email; }
        public void setEmail(string email) { this.email = email; }
        public string getDireccion() { return direccion; }
        public void setDireccion(string direccion) { this.direccion = direccion; }
        public bool isActivo() { return activo; }
        public void setActivo(bool activo) { this.activo = activo; }

        public override string ToString()
        {
            return nombre;
        }
    }
}

File: Modelo/Proveedor.py
class Proveedor:
    def __init__(self, id=0, nombre='', contacto='', telefono='', email='', direccion='', activo=False):
        self._id = id
        self._nombre = nombre
        self._contacto = contacto
        self._telefono = telefono
        self._email = email
        self._direccion = direccion
        self._activo = activo

    def getId(self):
        return self._id

    def setId(self, id):
        self._id = id

    def getNombre(self):
        return self._nombre

    def setNombre(self, nombre):
        self._nombre = nombre

    def getContacto(self):
        return self._contacto

    def setContacto(self, contacto):
        self._contacto = contacto

    def getTelefono(self):
        return self._telefono

    def setTelefono(self, telefono):
        self._telefono = telefono

    def getEmail(self):
        return self._email

    def setEmail(self, email):
        self._email = email

    def getDireccion(self):
        return self._direccion

    def setDireccion(self, direccion):
        self._direccion = direccion

    def isActivo(self):
        return self._activo

    def setActivo(self, activo):
        self._activo = activo

    def __str__(self):
        return self._nombre